raresGabriel01/home-assignment
# [ISSUE-3] Setup angular project

# Issue link
#3 
# Description
Setup an angular project with SASS via the Angular CLI (`ng new home-assignment`)
# How to test?

1. Open a terminal
2. `cd` into the root folder of this project
3. run `ng serve`
4. open your browser at the indicated address in the terminal
5. ensure a basic template project opens up

# Attachments
![image](https://github.com/raresGabriel01/home-assignment/assets/56756497/ee4db74f-20d3-48f4-99ec-133579a145e2)
